### DISCUSSION TITLE
Is this a TailwindCSS bug?

### DISCUSSION THREAD

**Discussion by felipetodev** [tailwindlabs/tailwindcss#12461] [Help]
I'm trying to do this in my Next.js project:
https://play.tailwindcss.com/coz07Km10V

But the result is different, is pushing the block idk why, here's the reproduction:
https://codesandbox.io/p/devbox/distracted-ben-53gsl4?file=%2Fapp%2Fpage.tsx%3A1%2C14

It seems that Next.js overwrite or change the native behavior of that grid property. Any ideas? 😢

**Comment by wongjn** [comment]
`<div/>` in HTML (i.e. the Tailwind Play) does not mean `<div></div>` as it does in React – it behaves as an open tag only, `<div>`.

Thus, to make the equivalent of the CodeSandbox project in Tailwind Play: https://play.tailwindcss.com/gGtlBynuCo

Or to make the equivalent of the Tailwind Play in the CodeSandbox: https://codesandbox.io/p/devbox/https-github-com-tailwindlabs-tailwindcss-discussions-12461-7nwt8f?file=%2Fapp%2Fpage.tsx%3A6%2C11

**Comment by felipetodev** [comment]
Wow, I hadn't considered that, thank you very much for the clarification @wongjn!
